### PR TITLE
Fix/ add decision as query parameter in pc console

### PR DESCRIPTION
**Pull request by xkopenreview**
'No Decision' is a fallback value rendered when the actual value is undefined(not found)
set it as the string value "No Decision" so that query filter can compare it the user input

### CODE REVIEW

File: openreview/conference/templates/programchairWebfield.js [hunk 1]
```diff
@@ -847,7 +848,7 @@ var calcMetaReviewCount = function(blindedNotes) {
 var calcDecisions = function(blindedNotes) {
   var finalDecisions = [];
   blindedNotes.forEach(function(note) {
-    if (note.details.decision) {
+    if (note.details.decision && note.details.decision !== 'No Decision') {
```
Comment (melisabok) [inline openreview/conference/templates/programchairWebfield.js:851]: I'm not sure when `note.details.decision` is a string

Comment (xkopenreview) [reply openreview/conference/templates/programchairWebfield.js:851]: when there's no decision i set it to the string 'No Decision'

Comment (melisabok) [reply openreview/conference/templates/programchairWebfield.js:851]: Could you instead create a decision object with 'No Decision'?

and there is no need to change openreview-web:

`var paperDecisions = _.find(submission.details.directReplies, ['invitation', getInvitationId(DECISION_NAME, submission.number)]) ?? [{ content: 'No Decision' }]`

Comment (melisabok) [reply openreview/conference/templates/programchairWebfield.js:851]: I would prefer to avoid that otherwise you need to check that everywhere.

Comment (xkopenreview) [reply openreview/conference/templates/programchairWebfield.js:851]: i think there was some difficulty with sorting when defaulting to object here but i can try again

Comment (melisabok) [reply openreview/conference/templates/programchairWebfield.js:851]: I don't think you need the second condition anymore, right?

Comment (melisabok) [inline openreview/conference/templates/programchairWebfield.js:851]:
```suggestion
    if (note.details.decision) {
```

**Comment by melisabok** [comment]
It looks good! I just made a suggested change. Please take a look.

File: openreview/conference/templates/programchairWebfield.js (hunk 1 above)
Comment (xkopenreview) [reply openreview/conference/templates/programchairWebfield.js]: thank it's reading a wrong property here but we can't remove this
removing this condition will cause the decision progress data in overview tab to be wrong where "No Decision" is considered as a final decision and the decision progress data will always be 100%

**Review by melisabok — APPROVED** [review]

**Pull request merged**